cmd/gotemplate/gotemplate.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"text/template"
+)
+
+// gotemplate is a simple CLI for text/template. It reads from stdin and writes to stdout.
+// Optional arguments are <key>=<value> pairs which can be used as {{.<key>}} to inject
+// the <value> for that key.
+//
+// Besides the default functions (https://pkg.go.dev/text/template#hdr-Functions),
+// gotemplate also implements:
+// - include <filename>: returns the content of that file as string
+// - indent <number of spaces> <string>: indent each line, including the first one, with spaces
+
+func main() {
+	kvs := make(map[string]string)
+	flag.Parse()
```

Comment:
do we need to call this if we have no flags?

Reply:
Not really. It adds `-h`, but that doesn't print anything useful. I've removed `flag`.